callbacks.go

```diff
@@ -150,14 +170,9 @@ func (exec *sidecarExecutor) KillTask(driver executor.ExecutorDriver, taskID *me
 	// Notify Mesos
 	exec.sendStatus(status, taskID)
 
-	// We have to give the driver time to send the message
-	time.Sleep(StatusSleepTime)
```

Comment:
FIY: I moved this into `exec.sendStatus`. We need to sleep each time we send a status. Maybe the new version of the Mesos API does this better.

Reply:
We don't need to sleep every time we send a status. We only have to sleep when sending a status _at_ _shutdown_.